third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -753,9 +753,10 @@ pub enum StatusCode {
     // Should never be committed on chain
     SPECULATIVE_EXECUTION_ABORT_ERROR = 2024,
     ACCESS_CONTROL_INVARIANT_VIOLATION = 2025,
+    // We tried to create resource with more than currently allowed number of DelayedFields
+    TOO_MANY_DELAYED_FIELDS = 2026,
 
     // Reserved error code for future use
-    RESERVED_INVARIANT_VIOLATION_ERROR_1 = 2026,
```

Comment:
@igor-aptos i think this still starts with _2?

Reply:
doesn't matter but sure :)

Reply:
I mean we can maybe keep like 1 reserved and not 5😄

Reply:
no, that is the amount you can utilize in a single release. 

basically these are reserved because it is a breaking change to add one that didn't exist in a previous release.

Reply:
Interesting, do you know why 5 then?

Reply:
I mean it is some number larger than what we expect amount of new errors in a single release we might add.